examples/data-solutions/data-playground/main.tf

```diff
@@ -104,10 +158,17 @@ resource "google_notebooks_instance" "playground" {
   install_gpu_driver = true
   boot_disk_type     = "PD_SSD"
   boot_disk_size_gb  = 110
+  disk_encryption    = try(local.service_encryption_keys.compute != null, false) ? "CMEK" : "GMEK"
+  kms_key            = try(local.service_encryption_keys.compute, null)
 
-  no_public_ip    = false
+  no_public_ip    = true
```

Comment:
[just double checking] no_public_ip is set to true, does that mean we won't be able to access the notebook's web UI unless we're connected to the same VPC network correct? Might be useful to include that as a quick comment in the README.

Reply:
We have no_proxy_access = false, you'll be able to connect to the UI